### DISCUSSION TITLE
Node is not executable

### DISCUSSION THREAD

**Discussion by BenigneDemetz** [zbirenbaum/copilot.lua#222] [Q&A]
Hello,

Sorry for my question, I just installed copilot.vim on neovim with lazynvim, it works but by searching solution for nvim cmp issue, I found copilot.lua.
I well set it up but when I try to :Copilot auth it says that my node file isn't executable. However it works with copilot.vim. 
i set the path for copilot.vim in the global variable and path for copilot.lua in the global var and the config.
![image](https://github.com/zbirenbaum/copilot.lua/assets/72600691/71d611a8-f039-45fe-8228-e2aabb4905b1)
![image](https://github.com/zbirenbaum/copilot.lua/assets/72600691/bd0b0d29-302f-4c3d-be4c-526d15d19f40)
![image](https://github.com/zbirenbaum/copilot.lua/assets/72600691/4ecee0ae-3ffd-4043-8563-03e6794a5903)

Thanks a lot for the help !

**Comment by BenigneDemetz** [comment]
sorry didn't precised it, the first screen is with .vim
two lasts are with .lua

**Reply by MunifTanjim** [reply]
What happens when you run `/usr/bin/node` in our terminal?

**Reply by BenigneDemetz** [reply]
It works perfectly and start node. Even full permissions doesn't work for copilot :(

**Reply by MunifTanjim** [reply]
What's the output of `:Copilot status` and `:Copilot version`?

**Comment by BenigneDemetz** [comment]
Errors due to unexecutable node :/
![image](https://github.com/zbirenbaum/copilot.lua/assets/72600691/439f4c75-f80a-475a-a966-de27bae51039)
![image](https://github.com/zbirenbaum/copilot.lua/assets/72600691/bf56559e-c74e-4866-aafb-d30830cbb969)
![image](https://github.com/zbirenbaum/copilot.lua/assets/72600691/116067b3-e426-41e0-93e8-f05b1419d518)

**Comment by theherk** [comment]
I'm in a similar situation here. If I run in neovide I get this error, but running nvim from terminal I do not. I thought maybe it was a runtime environment thing, but I don't think so.

`Copilot status` gives:

```
[Copilot] copilot_node_command(node) is not executable
[Copilot] Could not determine Node.js version
-----------
(exit code) 0
   (output) 
-----------
```

But `!node --version` gives `v21.2.0`. So it seems that it is found and executable by the neovim instance.
